scripts/tests/dfx_tests.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+export ORD_BITCOIN_RPC_USERNAME=ic-btc-integration
+export ORD_BITCOIN_RPC_PASSWORD="QPQiNaph19FqUsCrBRN0FII7lyM26B51fAMeBQzCb-E="
+
+kill_ssl_proxy() {
+    set +e
+    PID="$(ps aux | grep local-ssl-proxy | grep -v grep | awk '{print $2}')"
+    if [ -n "$PID" ]; then
+        kill -9 $PID
+    fi
+    set -e
+}
+
+set -e
+
+kill_ssl_proxy || true
+killall -9 icx-proxy || true
+dfx stop
+
+start_icx() {
+    killall icx-proxy
+    sleep 2
+    # Start ICX Proxy
+    dfx_local_port=$(dfx info replica-port)
+    icx-proxy --fetch-root-key --address 127.0.0.1:8545 --dns-alias 127.0.0.1:bd3sg-teaaa-aaaaa-qaaba-cai --replica http://localhost:$dfx_local_port &
+    sleep 2
+}
+
+cargo build --tests -p integration-tests --features dfx_tests
```

Comment:
Is this required? This will rebuild everything even if it was already built, you can simply require that the build script is executed before this one